### DISCUSSION TITLE
Claiming a free JCB package key is not working

### DISCUSSION THREAD

**Discussion by sexavier** [joomengine/Joomla-Component-Builder#973] [Q&A] [answered]
Hello. I watched two of the repositories listed for claiming a free JCB package key. I waited 12 hours and, when claiming here: https://www.joomlacomponentbuilder.com/claim-free-package-key im always getting this message: "Please insure that you only use your github username, that has the selected relationship with the selected repository on github.
If you have moments ago forked/watched/stared the repository, then you may need to wait an hour before the gitHub API is ready to validate your relationship with the repo."
I tried several times, with different repositories. What could be wrong? Thanks.
Sérgio

**Comment by Llewellynvdm** [comment, marked as answer]
JCB packages is currently not longer working, we are working on a new approach to this, yet the current Demo component that ship with JCB is in fact very advanced and should be enough to get going with any project, you should technically not need more.

Remember JCB is and **advance tool** for advance developer in Joomla/PHP.
